workshops/Tkinter/README.md

```diff
@@ -0,0 +1,135 @@
+---
+name:'Creating a UI with Tkinter.'
+description:'We will use python tkinter graphics to make User Interfaces. Its a nice world.'
+author: '@Bezlin Johnson'
+---
+
+# Final User Interface or UI.
+
+<img width="539" alt="Screenshot 2020-09-19 at 11 28 35 AM" src="https://user-images.githubusercontent.com/54895485/93660382-e47c0880-fa6b-11ea-8f9a-94f476991652.png">
+
+This will be the simple UI we will be making today. This is nice world we can even make cool softwares with this .
+Here is the [live_demo](https://repl.it/@bezlin/graphics#main.py) and [final_code](https://repl.it/@bezlin/graphics#main.py) (see src/app.js and public/app.css files).
+This workshop will take 50 minutes.
+![ezgif com-video-to-gif](https://user-images.githubusercontent.com/54895485/93660884-07f58200-fa71-11ea-8bbd-44b10c7624b5.gif)
+
+## Step 1 : SETUP
+
+We will be using Tkinter python library to develop the user Interface.
+
+### Setting up the environment for Tkinter
+
+[Repl.it](https://repl.it) is a simple yet powerful online IDE, Editor, Compiler, Interpreter, and REPL. Code, compile, run, and host in 50+ programming languages.So it is super awesome.
+
+To get started, go to https://repl.it/languages/Tkinter. If you cant find it just type it in the type box. Your coding environment will be ready in a few seconds!
+<img width="1440" alt="Screenshot 2020-09-18 at 5 39 06 PM" src="https://user-images.githubusercontent.com/54895485/93595705-dbd8f300-f9d5-11ea-8bfa-a26e407adeaf.png">
+
+## Step 2: Know the basics of Tkinter
```

Comment:
as a beginner this would be confusing for me.... I even I dont understand :D

Reply:
ok I will change that too

Reply:
coolio

Reply:
how can we edit this where should I edit it

Reply:
here: https://github.com/bezlin6mechminerz/hackclub/tree/patch-1/workshops/Tkinter